model/host/host.go

```diff
@@ -739,3 +744,53 @@ func CountInactiveHostsByProvider() ([]InactiveHostCounts, error) {
 	}
 	return counts, nil
 }
+
+// find all containers
+func (h *Host) FindAllContainers() ([]Host, error) {
+	query := db.Query(bson.M{
+		ContainerID: bson.M{"$exists": true},
+	})
+	hosts, err := Find(query)
+	if err != nil {
+		return nil, errors.Wrap(err, "Error finding containers")
+	}
+
+	return hosts, nil
+}
+
+// find all containers belonging to parent of current host
+func (h *Host) FindHostContainers() ([]Host, error) {
+	if !h.IsParent {
+		return nil, errors.New("Host is not a parent of any containers")
+	}
+	query := db.Query(bson.M{
+		ParentID: h.ParentID,
+	})
+	hosts, err := Find(query)
+	if err != nil {
+		return nil, errors.Wrap(err, "Error finding containers")
+	}
+
+	return hosts, nil
+}
+
+// find parent of container
```

Comment:
our documentation convention would be for this to say "FindParentOfContainer finds the parent of this container". Yes, it's sometimes redundant. I'd also document the error scenarios ("error if this is not a container"). Please modify this one and the others.